lightning/src/ln/peer_handler.rs

```diff
@@ -1167,10 +1184,15 @@ mod tests {
 
 	fn establish_connection<'a>(peer_a: &PeerManager<FileDescriptor, &'a test_utils::TestChannelMessageHandler>, peer_b: &PeerManager<FileDescriptor, &'a test_utils::TestChannelMessageHandler>) {
 		let secp_ctx = Secp256k1::new();
-		let their_id = PublicKey::from_secret_key(&secp_ctx, &peer_b.our_node_secret);
-		let fd = FileDescriptor { fd: 1};
-		peer_a.new_inbound_connection(fd.clone()).unwrap();
-		peer_a.peers.lock().unwrap().node_id_to_descriptor.insert(their_id, fd.clone());
+		let a_id = PublicKey::from_secret_key(&secp_ctx, &peer_a.our_node_secret);
+		//let b_id = PublicKey::from_secret_key(&secp_ctx, &peer_b.our_node_secret);
```

Comment:
Eh, remove unused code.

Reply:
this code is actually directly taken from Matt's commit. If I remove it, will it cause you merging headaches, @TheBlueMatt?

Reply:
Oops, sorry, that was cause I was just playing and testing, you can remove it :).